lib/order.rb

```diff
@@ -0,0 +1,84 @@
+require 'customer.rb'
+require 'csv'
+
+class Order
+
+  attr_reader :id, :products, :customer, :fulfillment_status
+
+  def initialize(id, products, customer, fulfillment_status = :pending)
+    @id = id
+    @products = products
+    @customer = customer
+    if [:pending, :paid, :processing, :shipped, :complete].include?(fulfillment_status)
+      @fulfillment_status = fulfillment_status
+    else
+      raise ArgumentError, "Invalid fulfillment status."
```

Comment:
It's helpful for debugging to include the invalid argument in the message for your `ArgumentError`s:

```suggestion
      raise ArgumentError, "Invalid fulfillment status: #{fulfillment_status}"
```